sdk/servicebus/service-bus/.eslintrc.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
I'd like to poke at this and see if I can get the list of rules reduced - ideally we should be as close to `eslint:recommended` as possible, right?

Also, if we think this is a good eslint config for all of our projects, is it possible to store this in the root of the repo and then reference it in each package like the following example:

.eslintrc.json:
```
{
    "extends": ["azure-sdk-for-js"]
}
```

I wouldn't do the symlinking thing here, since each package may need to actually override some of our common rules (unlike Prettier which is designed to be consistent across everything that uses it).

Reply:
@bsiegel  The eslint rules in the PR are basically conversion of the existing tslint rules configured for the package. I did cross-verify to see if i have any unnecessary conflicting rules with those covered by prettier and removed those. But whether we would want to revise for the other rules and about standardizing across all packages, I will discuss this with @bterlson

Reply:
Sounds good! I'm not 100% sure of the mechanism, but I'm reasonably sure we can keep a central copy of our "best practice" lint rules, and then `extend` from them in each library's .eslintrc.json, overriding or adding specific rules as needed for the library.

Regardless, that should definitely be its own task, I wouldn't argue you should try to do that in this PR.